src/guides/bioinformatics/filtering-and-subsampling.rst

```diff
@@ -23,20 +24,101 @@ different types of analysis. A simple example use of this command would be
      --output-sequences filtered_sequences.fasta \
      --output-metadata filtered_metadata.tsv
 
-This command will select all sequences with collection date in 2012 or later.
-The filter command has a large number of options that allow flexible filtering
-for many common situations. One such use-case is the exclusion of sequences that
-are known to be outliers (e.g. because of sequencing errors, cell-culture
-adaptation, ...). These can be specified in a separate text file (e.g.
-``exclude.txt``):
+There are several options that allow flexible filtering for many common
+situations. Options can be divided into the following categories:
+
+- **Metadata-based** options work with information available from
+  ``--metadata``.
+- **Sequence-based** options work with information available from
+  ``--sequences`` or ``--sequence-index``.
+- **Standard** options work by selecting or dropping sequences that match
```

Comment:
Would it be fair to divide this set of options into 3 groups: "standard filtering" (preliminary filtering?), "subsampling", and "force-inclusion", and then organise the docs that way? Something like:

**preliminary filtering steps**

**subsampling steps**

**force-inclusion steps**

**alternate approach: subsampling using multiple augur filter commands**

Reply:
Oh that makes sense! I'll try it. Do you think it's still worth distinguishing metadata vs. sequence based filters here? They're already distinguished on the [augur filter docs](https://docs.nextstrain.org/projects/augur/en/latest/usage/cli/filter.html), so I'm thinking no need to add it here.

Reply:
> P.S. I was going to suggest going all out and linking each argument in the table to the corresponding entry in the the augur filter docs, but there's not anchors for each argument.

+1 for https://github.com/nextstrain/augur/issues/1458

Reply:
I've [updated](https://github.com/nextstrain/docs.nextstrain.org/compare/96f68da125c4c34d05b5f5fbf41c22bc4b399179..62e01e51d3523b9fff7c633792f83dfaaf1a5efc) the "Preliminary & force-inclusive" section header to "Preliminary & force-inclusive sampling" to fit better with other section headers.

Reply:
After bikeshedding with myself I've determined that "Preliminary & force-inclusive **selection**" contrasts better with **subsampling**. [Updated](https://github.com/nextstrain/docs.nextstrain.org/compare/62e01e51d3523b9fff7c633792f83dfaaf1a5efc..5779a70fc3b8b5cc4c6bcdbf4b01a7360e6dd236). I'm going to stop arguing with myself and merge now.